### ISSUE TITLE
Package intoo required but no longer available on CRAN

### ISSUE THREAD

**Issue by tmalsburg** [vasishth/Freq_CogSci#27]
See [here](https://github.com/vasishth/Freq_CogSci/blob/master/index.Rmd#L103).  As a result, it is currently impossible to compile the book (unless someone still has `intoo` installed on their system).

**Comment by vasishth** [comment]
Get the archived package and install using cmd install

**Comment by tmalsburg** [comment]
A couple of other packages that are used but no longer available on CRAN: 
- barsurf
- bivariate
- intoo
- kubik

To install them:
``` sh
wget https://cran.r-project.org/src/contrib/Archive/bivariate/bivariate_0.7.0.tar.gz
wget https://cran.r-project.org/src/contrib/Archive/intoo/intoo_0.4.0.tar.gz
wget https://cran.r-project.org/src/contrib/Archive/barsurf/barsurf_0.7.0.tar.gz
wget https://cran.r-project.org/src/contrib/Archive/kubik/kubik_0.3.0.tar.gz
```

The package `lingpsych` is also used (but not available on CRAN):
``` sh
R -e 'devtools::install_github("vasishth/lingpsych")'
```

**Comment by tmalsburg** [comment]
I had to compile the book dozens of time only to find out, after waiting a couple of minutes, that yet another package was missing.  It would be good to either load all packages at the beginning so the compilation fails immediately if something is missing, or to have a separate script that installs all dependencies.

**Comment by vasishth** [comment]
yes, thanks. i will do this. am leaving this open for this reason.

**Comment by tmalsburg** [comment]
It might also be good to have a Makefile with a rule for cleaning the repository from any generated files (`make clean`).  This way, you can establish that the book compiles from scratch.

**Comment by vasishth** [comment]
Good idea. I will test workflow for installing it on my new laptop, where I have to start from scratch. Will set up a Makefile.
